AdaptiveCards/templating/index.md

```diff
@@ -12,7 +12,16 @@ We're excited to share a preview of new tools that will help you **create**, **r
 
```

Comment:
Per discussion today, perhaps add a Troubleshooting section here in FAQ format, to aid in common user errors (Eg errors around using the old syntax with hte newer package. )